docs/_snippets/override-compose-story-test.md

```diff
@@ -1,144 +1,37 @@
-```js filename="Form.test.js|jsx" renderer="react" language="js" tabTitle="compose-stories"
-import { composeStories } from '@storybook/react';
+```js filename="Form.test.js|ts" renderer="common" language="js" tabTitle="compose-stories"
+// Replace your-renderer with the renderer you are using (e.g., react, vue3, angular, etc.)
```

Comment:
I wonder if we should do `react, vue3 or svelte` or we leave the `etc.` for the future, when more renderers are supported. Actually there is also `composeStories` in `@storybook/nextjs`
```suggestion
// Replace your-renderer with the renderer you are using (e.g., react, vue3, svelte, etc.)
```